### PR TITLE
feat(plugin): select address component to send scene added

### PR DESCRIPTION
**Pull request by OlehPerehuda**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Send/SelectAddress/AddressInput/index.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { connect, ConnectedProps } from 'react-redux'
+import {
+  getErc20Data,
+  getEthData
+} from 'blockchain-wallet-v4-frontend/src/components/Form/SelectBoxEthAddresses/selectors'
+import { AvailableCoins } from 'blockchain-wallet-v4-frontend/src/scenes/plugin/Send'
+import AddressInput from 'blockchain-wallet-v4-frontend/src/scenes/plugin/Send/SelectAddress/AddressInput/template.succes'
```
Comment (NikolaiYurchenko) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Send/SelectAddress/AddressInput/index.tsx]: success?

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Send/SelectAddress/template.success.tsx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  // changes and validates wallet address
+  const changeAddress = () => {
+    changeStep(AvailableSteps.FirstStep)
+  }
+
+  const goBack = () => {
+    history.goBack(-1)
+  }
+
+  const isFromCustody = from && from.type === 'CUSTODIAL'
```
Comment (TarasPotsikaylo) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Send/SelectAddress/template.success.tsx]: we don`t have custodial account 
why do we need this check? 
Can we add _includeCustodial={false}_ ?

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Send/index.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+import { actions } from 'data'
+
+import SelectAddress from './SelectAddress'
+
+// defines available steps to send amount
+export enum AvailableSteps {
+  SelectAddress = 0,
+  FirstStep = 1,
+  SecondStep = 2
+}
```
Comment (TarasPotsikaylo) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Send/index.tsx]: You don`t need to add =0, =1 etc
Just 
```
export enum AvailableSteps {
  SelectAddress,
  FirstStep,
  SecondStep
}
```

Comment (OlehPerehuda) [reply packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Send/index.tsx]: yea, really good point

**Review by VitaliiShpital — APPROVED** [review]

**Pull request merged**